src/app_service/service/general/service.nim

```diff
@@ -100,4 +100,5 @@ QtObject:
       let response =  status_general.backupData()
       return response.result.getInt
     except Exception as e:
-      error "error: ", procName="backupData", errName = e.name, errDesription = e.msg
\ No newline at end of file
+      error "error: ", procName="backupData", errName = e.name, errDesription = e.msg
+    
```

Comment:
Stray empty line?

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline ;)

Reply:
And btw, QtC can help here :)

![image](https://github.com/status-im/status-desktop/assets/5377645/3bec21ed-c81c-472d-8a11-d3bf03ae0a12)

Reply:
Pretty sure VSCode has something like that too